### DISCUSSION TITLE
Invalid Configuration via cdn

### DISCUSSION THREAD

**Discussion by tanglu1323123** [vercel/community#3512] [Help] [answered]
### Summary

Hi,
I successfully used the vercel, and it works well !
I set a cdn for my domain lsat weekend,my project still works well, but the vercel shows Invalid Configuration and email me update or remove.

I just changed the 'waline.tanglu.me cname cname.vercel-dns.com' to 'waline.tanglu.me cname mycdn.com, then cdn to the cname.vercel-dns.com'.

I want to know if my project will be removed by vercel, thanks!
 

### Steps to Reproduce

![image](https://img.tanglu.me/2023/08/06/4uv1d.webp)

**Comment by fabricio-fn** [comment, marked as answer]
No, it will only be deleted if you delete it yourself in settings
